### PR TITLE
Add scrollwheel sidebar scrolling support

### PR DESCRIPTION
**Pull request by hazelnot**
Just Iran's PR (#383) but updated to also add scrollwheel support to TD

### CODE REVIEW

File: redalert/conquer.cpp [hunk 1]
```diff
@@ -4233,7 +4235,7 @@ bool Force_CD_Available(int cd)
 #endif
 
 #ifdef FRENCH
-                sprintf(buffer, "Ins�rez le %s", _cd_name[cd]);
+                sprintf(buffer, "InsŠrez le %s", _cd_name[cd]);
```
Comment (giulianobelinassi) [inline redalert/conquer.cpp]: I don't think this should be here. If that is a spelling error fix it should be in another commit.

**Review by giulianobelinassi — CHANGES_REQUESTED** [review]

File: common/settings.cpp [hunk 2]
```diff
@@ -37,6 +38,7 @@ void SettingsClass::Load(INIClass& ini)
     */
     Mouse.RawInput = ini.Get_Bool("Mouse", "RawInput", Mouse.RawInput);
     Mouse.Sensitivity = ini.Get_Int("Mouse", "Sensitivity", Mouse.Sensitivity);
+    Options.MouseWheelScrolling = ini.Get_Bool("Options", "MouseWheelScrolling", Video.FrameLimit);
```
Comment (tomsons26) [inline common/settings.cpp]: Wrong default being used

File: common/settings.cpp [hunk 3]
```diff
@@ -74,6 +76,7 @@ void SettingsClass::Save(INIClass& ini)
     */
     ini.Put_Bool("Mouse", "RawInput", Mouse.RawInput);
     ini.Put_Int("Mouse", "Sensitivity", Mouse.Sensitivity);
+    ini.Put_Bool("Options", "MouseWheelScrolling", Video.FrameLimit);
```
Comment (tomsons26) [inline common/settings.cpp]: Wrong default being used

File: redalert/conquer.cpp (hunk 1 above)
Comment (hazelnot) [reply redalert/conquer.cpp]: It's not a spelling fix, the compare tool wouldn't let me save the file if I didn't "fix" that, so I just copied what Sublime displayed that line as. It's some sort of encoding error, but thanks, I'll fix it

File: common/settings.cpp (hunk 2 above)
Comment (hazelnot) [reply common/settings.cpp]: That isn't my code, so I don't know what I'd have to change about it

File: common/settings.cpp (hunk 3 above)
Comment (hazelnot) [reply common/settings.cpp]: That isn't my code, so I don't know what I'd have to change about it

File: redalert/conquer.cpp (hunk 1 above)
Comment (hazelnot) [reply redalert/conquer.cpp]: I tried to copy the text from the original and now it displays as "Insrez" in the browser and `Ins<0x8a>rez` in Sublime. It's supposed to be an "é" but as it stands I have no idea how to revert it or what the original encoding for that was

File: common/settings.cpp (hunk 3 above)
Comment (giulianobelinassi) [reply common/settings.cpp]: You are outputing the FrameLimit variable to MouseWheelScrolling text in .ini instead of the MouseWheelScrolling variable itself.

File: common/settings.cpp (hunk 2 above)
Comment (giulianobelinassi) [reply common/settings.cpp]: Options.MouseWheelScrolling = ini.Get_Bool("Options", "MouseWheelScrolling", Options.MouseWheelScrolling);
